browser/js/search-popup/search-popup.js

```diff
@@ -91,7 +87,7 @@ function DialogCtrl ($timeout, $q, $scope, $mdDialog, $rootScope, Users) {
       var lowercaseQuery = angular.lowercase(query);
 
       return function filterFn(employee) {
-        return (employee.value.indexOf(lowercaseQuery) === 0);
+        return (employee.value.indexOf(lowercaseQuery) !== -1);
```

Comment:
A new friend for you [es6 `includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes).